hangups/http_utils.py

```diff
@@ -46,6 +46,9 @@ def fetch(method, url, params=None, headers=None, cookies=None, data=None,
         else:
             error_msg = None
             break
+        finally:
+            if 'res' in locals():
+                res.close()
```

Comment:
We probably want to switch to using `ClientSession` at some point since that's the recommended interface in aiohttp now. `hangups.Client` does have a `Connector` [which it attempts to close here](https://github.com/tdryer/hangups/blob/2a72f30bc89e8e385e713379fd46ae7531be8ba7/hangups/client.py#L142).